backend/internal/controllers/img.go

```diff
@@ -0,0 +1,74 @@
+package controllers
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	"github.com/TritonSE/words-alive/internal/database"
+	"github.com/TritonSE/words-alive/internal/utils"
+
+	"github.com/go-chi/chi"
+)
+
+type ImgController struct {
+	Image database.ImgDatabase
+}
+
+var allowedTypes = map[string]bool{
+	"image/png":  true,
+	"image/jpeg": true,
+}
```

Comment:
how dare you, this is so shamefully wasteful, you could save TWO WHOLE BYTES by converting this to a `map[string]struct{}`, and using `if _, found := allowedTypes[key] {`

kidding, this is fine, you can leave it if you want, just a nice pro tip in case you ever need a much larger set with millions of entries